scripts/bank/export-bank-receipts.php

```diff
@@ -34,7 +34,7 @@
 // Test if batch mode
 if (substr($sapi_type, 0, 3) == 'cgi') {
 	echo "Error: You are using PHP for CGI. To execute ".$script_file." from command line, you must use PHP for CLI mode.\n";
-	exit(-1);
+	exit(255);
```

Comment:
Currently all scripts, templates are using -1, -2, ... for error code. So for consistency we should keep this.
What is the test that fails on windows ?